catboost/private/libs/options/catboost_options.cpp

```diff
@@ -667,7 +673,7 @@ void NCatboostOptions::TCatBoostOptions::Validate() const {
         // we may adjust non-set BoostFromAverage in data dependant tuning
         CB_ENSURE(EqualToOneOf(lossFunction, ELossFunction::RMSE, ELossFunction::Logloss,
             ELossFunction::CrossEntropy, ELossFunction::Quantile, ELossFunction::MAE, ELossFunction::MAPE,
-            ELossFunction::MultiRMSE),
+            ELossFunction::MultiRMSE, ELossFunction::MultiRMSEWithMissingValues),
             "You can use boost_from_average only for these loss functions now: " <<
             "RMSE, Logloss, CrossEntropy, Quantile, MAE, MAPE or MultiRMSE.");
```

Comment:
add MultiRMSEWithMissingValues to message

Reply:
Strange. Is already there.

Reply:
At 677-678 lines message "You can use boost_from_average ... , MultiRMSE **or MultiRMSEWithMissingValues**."